tests/local.sh

```diff
@@ -24,4 +24,5 @@ remote SKIP_PREREQUISITES="${SKIP_PREREQUISITES}" ./tests/scripts/prerequisites.
 remote \
     PROJECT="${PROJECT}" \
         TARGET_DRIVER_VERSION="${TARGET_DRIVER_VERSION}" \
+        OPERATOR_OPTIONS=\"${OPERATOR_OPTIONS}\" \
```

Comment:
Enhancement: Pass a single argument containing all enviroment variables  and parse it on the target to export them